content/es/docs/tasks/debug-application-cluster/debug-init-containers.md

```diff
@@ -0,0 +1,128 @@
+---
+title: Depurar Contenedores de Inicialización
+content_template: templates/task
+---
+
+{{% capture overview %}}
+
+Esta página muestra cómo investigar problemas relacionados con la ejecución
+de los contenedores de inicialización (init containers). Las líneas de comando del ejemplo de abajo
+se refieren al Pod como `<pod-name>` y a los Init Containers como `<init-container-1>` e
+  `<init-container-2>` respectivamente.
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+* Deberías estar familizarizado con el concepto de [Init Containers](/docs/concepts/abstractions/init-containers/).
+* Deberías conocer la [Configuración de un Init Container](/docs/tasks/configure-pod-container/configure-pod-initialization/#creating-a-pod-that-has-an-init-container/).
+
+{{% /capture %}}
+
+{{% capture steps %}}
```

Comment:
This looks invalid. I'll propose a pull request to fix it.

Reply:
I can do it, this comes from a really old branch. https://github.com/kubernetes/website/pull/13981

Reply:
Done in https://github.com/kubernetes/website/pull/22494